### PR TITLE
Containerization of Dashboard

### PR DESCRIPTION
**Pull request by MVarshini**
CHAOS-787

Containerization of Dashboard

### CODE REVIEW

File: containers/build_own_image-README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Building your own Kraken dashboard image
+- Git clone the Kraken repository using git clone https://github.com/krkn-chaos/krkn-dashboard.git
+- Execute ` podman build -t <new_image_name>:latest . ` in the containers directory within krkn-dashboard to build an image from a Dockerfile.
```
Comment (chaitanyaenr) [inline containers/build_own_image-README.md:3]: The command needs to be: $ podman build -t <new_image_name>:latest -f containers/dockerfile to be able to copy the source code into the container.

File: containers/build_own_image-README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Building your own Kraken dashboard image
+- Git clone the Kraken repository using git clone https://github.com/krkn-chaos/krkn-dashboard.git
+- Execute ` podman build -t <new_image_name>:latest -f containers/dockerfile ` in the containers directory within krkn-dashboard to build an image from a Dockerfile.
+- Once the build is completed, run ` docker image ls ` to list all the images. The newly created image can be found here.
```
Comment (chaitanyaenr) [inline containers/build_own_image-README.md:4]: s/docker/podman

Comment (MVarshini) [reply containers/build_own_image-README.md]: Updated

**Review by chaitanyaenr — APPROVED** [review]
LGTM!

**Pull request merged**